### ISSUE TITLE
Checkbox - Can't select 

### ISSUE THREAD

**Issue by jamesj-0** [fac19/week3-JICG#41]
When navigating the page with my keyboard I was unable to hit the enter key to select the checkbox.

We had the same issue! Here's a code snippet of the fix (sorry for the spoiler!)

        checkbox.addEventListener("keypress", function(e){
            if(e.which === 13){
                this.checked = !this.checked;
            }
        });
